src/python/datapreprocessor/datapreprocessor/personclient.py

```diff
@@ -0,0 +1,33 @@
+from ipersonclient import IPersonClient
+
+class PersonClient(IPersonClient):
+    def get_all(self):
+        response = self._get(self._build_url('people'))
+        items = response.json()['data']
+        return items
+
+
+    def get_by_name(self, person_name):
+        people = self.get_all()
+        person = filter((lambda p: p['attributes']['name'] == person_name), people)
+        if person:
+            return person[0]
+        else:
+            return None
+
+
+    def add(self, person_name):
+        if self.get_by_name(person_name):
+            return None
+        else:
+            data = {'data': {'type': 'people', 'attributes': {'name': person_name}}}
+            response = self._post(self._build_url('people'), data)
+            return response
+
+
+    def delete(self, person_name):
+        person = self.get_by_name(person_name)
+        if person:
+            return self._delete(person['links']['self'])
```

Comment:
And again, this could probably just return True on success and False if there was an error and the person wasn't removed.
If the person's not present then I'm not sure if that would be True (since the desired state of having the  person not be there any more has been achieved) or if some sort of person not found Exception should be raised. I'd probably lean towards just returning True to begin with and if the other situation is needed later adding an `error_on_missing_person` argument to the function that enables the other version.